projects/nftfi/index.js

```diff
@@ -1,15 +1,31 @@
 const { sumTokensExport, } = require('../helper/unwrapLPs');
 
 // Vaults
-const v2 = "0xf896527c49b44aAb3Cf22aE356Fa3AF8E331F280";
-const v2_1 = "0x8252Df1d8b29057d1Afe3062bf5a64D503152BC8";
-const offers_ADDR = "0xe52cec0e90115abeb3304baa36bc2655731f7934";
-const bundles = "0x16c583748faed1c5a5bcd744b4892ee6b6290094";
+const loan_offer_v2 = "0xf896527c49b44aAb3Cf22aE356Fa3AF8E331F280";
+const loan_offer_v2_1 = "0x8252Df1d8b29057d1Afe3062bf5a64D503152BC8";
+const loan_offer_v2_2 = "0xD0C6e59B50C32530C627107F50Acc71958C4341F";
+
+const loan_collection_offer_v2_1 = "0xe52cec0e90115abeb3304baa36bc2655731f7934";
+const loan_collection_offer_v2_2 = "0xD0C6e59B50C32530C627107F50Acc71958C4341F";
```

Comment:
is this address correct? looks like duplicate of `loan_offer_v2_2`

Reply:
that is indeed a mistake, the correct address for `loan_offer_v2_2` is `0xd0a40eB7FD94eE97102BA8e9342243A2b2E22207`

the one for `loan_collection_offer_v2_2` is good

Reply:
Made a fix in this PR, sorry for making a mess:)
https://github.com/DefiLlama/DefiLlama-Adapters/pull/8038